### DISCUSSION TITLE
has any got the Langflow to GCP install to work?

### DISCUSSION THREAD

**Discussion by pleabargain** [langflow-ai/langflow#473] [Q&A]
I keep getting 
```error device out of space```

https://youtu.be/CY6pbP4hLyM

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.

Check it out here:
https://docs.langflow.org/deployment-gcp
